### ISSUE TITLE
Typo in Banner "Zeebee" vs. "Zeebe"

### ISSUE THREAD

**Issue by pihme** [zeebe-io/zeebe#4111]
**Describe the bug**
There is a typo in the banner

**To Reproduce**
Launch Broker or Gateway

**Comment by Zelldon** [comment]
btw could we also do it more bold like this for example:
https://devops.datenkollektiv.de/banner.txt/index.html

```
  ______  ______   ______   ____    ______ 
 |___  / |  ____| |  ____| |  _ \  |  ____|
    / /  | |__    | |__    | |_) | | |__   
   / /   |  __|   |  __|   |  _ <  |  __|  
  / /__  | |____  | |____  | |_) | | |____ 
 /_____| |______| |______| |____/  |______|
```

**Comment by pihme** [comment]
Which font did you use?

**Comment by Zelldon** [comment]
I used for this example 'big'

**Comment by npepinpe** [comment]
Fun site; I like alpha, blocks, isometric1, starwars, swampland, and varsity. Though in the end as long as it's readable, I'm fine with anything - I think I just like the isometric ones :sweat_smile:

**Comment by pihme** [comment]
Doom is also quite readable ;-)

**Comment by Zelldon** [comment]
Maybe we can have a pool in the team meeting

**Comment by pihme** [comment]
Fixed by #4025

The issue of the font will be addressed by #4120
